README.md

```diff
@@ -49,13 +49,13 @@ Before activating a server, there's the option to test it through a
 basic-auth password-protected link in the "Authenticated Link" section
 (the username and password are embedded in the link).
 
-Sometimes Open edX playbook fails, and then you need to read the log,
```

Comment:
These changes are outside the scope of BB-804. 
Please remove them from this PR.

Reply:
Ah, I've just asked because you also updated this block of text here, which isn't related to the cleanup.
```
Sometimes Open edX playbook fails, and then you need to read the log, which is
shown in real-time in the web console.  You can fix the settings and then spawn
another server.  Failed and old inactive servers and MySQL databases are
automatically cleaned up after three days.  An important feature is that Ocim
*grants SSH access* to members of a configurable GitHub organization, so you
can always SSH to an appserver's IP, *even if Open edX's deployment failed*,
and then debug it.  You can use your GitHub username and key.
```

Reply:
There are changes that are related to the PR:

 * "old inactive server **and MySQL databases** are"
 * "automatically cleaned up after **three days**" rather than "**some configurable amount of days**".

But then I re-flowed the text for better readability, which isn't related.

Would you prefer if I reverted these changes and made them in a separate PR?

Reply:
@jamestait This paragraph is talking about Ocim automatic cleanup of Appservers and isn't related to the integration cleanup.
If you leave an old Appserver or a deployment through Ocim fails, it'll be automatically cleaned after 3 days, and this process isn't related to the CI runs :grin:

Reply:
Ah, got it! I thought it was actually the CI cleanup process that was responsible for this, sorry! In that case, I'll revert this change.